test/integration/routes/client.js

```diff
@@ -0,0 +1,163 @@
+const { get, set } = require('@0ti.me/tiny-pfp');
```

Comment:
The fact that all of these per-route integration tests are basically exactly the same is concerning to me.

Integration tests probably normally duplicate a lot of logic, but these are pretty egregious duplications of code. This is especially annoying if we add a route and have to add it to _all_ route files or something, for example.

I don't really have a solution to this problem, though. I guess we could build a `routeTestCreator` function which dynamically generates tests, but if I'm not mistaken, that's often regarded as extremely bad practice. I think my tests already tend to have too much complexity as they are.

Anyway, curious what you think about this.